apps/web/src/ee/clerk/providers/EnterpriseAuthProvider.tsx

```diff
@@ -200,5 +200,6 @@ const toOrganizationEntity = (clerkOrganization: OrganizationResource): IOrganiz
     defaultLocale: clerkOrganization.publicMetadata.defaultLocale,
     domain: clerkOrganization.publicMetadata.domain,
     productUseCases: clerkOrganization.publicMetadata.productUseCases,
+    language: clerkOrganization.publicMetadata.language,
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** ‏ consider pluralization (just like in ee)

Reply:
I'm not author of this specific change so I just mirrored the questionnaire as is to the EE version.
We are using this data in tracking so maybe this exact key name is used somewhere in tracking to build some graph or maybe even something else I'm not aware of, so I try to avoid making changes that are not 100% necessary to avoid introducing potential bugs that could delay release.